src/main/resources/db/changelog/changelog-master.xml

```diff
@@ -23,4 +23,6 @@
   <include file="/changes/v3.0/add-authority-source-protocol.xml" relativeToChangelogFile="true"/>
   <include file="/changes/v3.0/add-authority-source-file-optimistic-locking.xml" relativeToChangelogFile="true"/>
   <include file="/changes/v3.0/update-auto-linking_enabled.xml" relativeToChangelogFile="true"/>
+  <include file="/changes/v4.0/add-additional-fields-to-authority.xml" relativeToChangelogFile="true"/>
+  <include file="/changes/v4.0/add-additional-fields-to-authority-archive.xml" relativeToChangelogFile="true"/>
```

Comment:
Please use a version from pom.xml (v3.1) instead of v4.0

Reply:
done